packages/puppeteer-core/src/bidi/core/BrowsingContext.ts

```diff
@@ -417,6 +417,18 @@ export class BrowsingContext extends EventEmitter<{
     });
   }
 
+  @throwIfDisposed<BrowsingContext>(context => {
+    // SAFETY: Disposal implies this exists.
+    return context.#reason!;
+  })
+  async setCacheBypass(enabled: boolean): Promise<void> {
+    // @ts-expect-error not in BiDi types yet.
+    await this.#session.send('browsingContext.setCacheBypass', {
```

Comment:
Our command is in the network module
```suggestion
    await this.#session.send('network.setCacheBypass', {
```